File: represent/bench/c_flat/SpecGenBench_0317__SpecGenBench__common__IsPalindrome__IsPalindrome.c
#include <string.h>

int isPalindrome(const char * s);

int isPalindrome(const char * s) {

        int n = ((int)strlen(s));

        for(int i = 0; i < ((int)strlen(s)); i++) {
            if(s[i) != s[n - 1 - i)) {
                return 0;
            }
        }

        return 1;
}

File: represent/bench/java_flat/SpecGenBench_0317__SpecGenBench__common__IsPalindrome__IsPalindrome.java
class IsPalindrome {
    public static boolean isPalindrome(String s) {
        int n = s.length();

        for(int i = 0; i < s.length(); i++) {
            if(s.charAt(i) != s.charAt(n - 1 - i)) {
                return false;
            }
        }

        return true;
    }
}
